docs/tools/lab/view-xdr/xdr-to-json.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "XDR to JSON"
+sidebar_label: XDR to JSON
+description: "The XDR to JSON converter transforms base-64 encoded XDR data into readable JSON format."
+sidebar_position: 0
+---
+
+# XDR to JSON
+
+To access the [XDR to JSON](https://lab.stellar.org/xdr/view) page, select "View XDR" from the left-hand navigation menu, then choose the "XDR to JSON" option from the submenu.
+
+![Lab: XDR to JSON page](/assets/lab/lab-xdr-to-json.png)
+
+## Inputting XDR Data
+
+On this page, you can enter a base-64 encoded [XDR](../../../learn/fundamentals/data-format/xdr.mdx) into the input field located at the top. Alternatively, you can get a sample XDR by selecting the "fetch the latest transaction" link positioned below the input field.
+
+## Understanding the Output
+
+After loading a valid XDR (for example, by fetching the latest transaction), the interface displays three key components:
+
+1. **Transaction Hash** – A read-only field containing the transaction hash, which can be copied for use in other tools, such as retrieving transaction details via [API Explorer > RPC Methods > getTransaction](https://lab.stellar.org/endpoints/rpc/get-transaction).
```

Comment:
I would like to add a tip on the limitation of get transaction rpc url - (7 days limitation)